bno055:
  ros__parameters:
    ros_topic_prefix: "bno055/"
    connection_type: "uart"
    uart_port: "/dev/ttyUSB0"
    uart_baudrate: 115200
    uart_timeout: 0.1
    data_query_frequency: 100
    calib_status_frequency: 0.1
    frame_id: "bno055"
    operation_mode: 0x0C
    placement_axis_remap: "P2"
    acc_factor: 100.0
    mag_factor: 16000000.0
    gyr_factor: 900.0
    acc_offset: [0xFFEC, 0x00A5, 0xFFE8]
    mag_offset: [0xFFB4, 0xFE9E, 0x027D]
    gyr_offset: [0x0002, 0xFFFF, 0xFFFF]
